java/test/jmri/util/swing/JemmyUtil.java

```diff
@@ -35,6 +35,7 @@ static public void pressDialogButton(String dialogTitle, String buttonName) {
         JButtonOperator jbo = new JButtonOperator(jdo, buttonName);
         // Click button
         jbo.push();
+        jdo.waitClosed();
     }
```

Comment:
The button here doesn’t necessarily result in closing the dialog window.  Please remove the waitClosed from this method.